contracts/pair_anchor/src/lib.rs

```diff
@@ -0,0 +1,15 @@
+pub mod contract;
+pub mod state;
+
+pub mod error;
+
+mod response;
+
+#[cfg(test)]
+mod testing;
+
+#[cfg(test)]
+mod mock_querier;
+
+// #[cfg(integration)]
```

Comment:
this should also be changed some way, but had some issues on running the integration tests